cluster/gce/util.sh

```diff
@@ -715,6 +721,10 @@ function create-network() {
     # The network needs to be created synchronously or we have a race. The
     # firewalls can be added concurrent with instance creation.
     gcloud compute networks create --project "${PROJECT}" "${NETWORK}" --mode=auto
+  else
+    PREEXISTING_NETWORK=true
+    PREEXISTING_NETWORK_MODE="$(gcloud compute networks list ${NETWORK} | tail -n -1 | tr -s ' ' | cut -f2 -d' ' || true)"
```

Comment:
Depending on the human readable output of gcloud has bit us in the past. Can you instead pass in a format string?